### PR TITLE
Requirements cleanup

### PR DESCRIPTION
**Pull request by slowr**
### Description of PR

- Updated dependencies and also removed unused ones. 
- Added commit after the select query in the testing script.

What component(s) does this PR affect?

- [x] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [x] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [x] This change requires a change in the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by slowr** [comment]
@vkotronis I removed dependencies that are installed through other to avoid upgrading wrongly a inner-dependency that will not support the parent dependency.

File: backend/requirements.txt [hunk 1]
```diff
@@ -1,18 +1,10 @@
-amqp==2.3.2
+kombu==4.2.2.post1
+ruamel.yaml==0.15.87
+redis==3.1.0
+psycopg2-binary==2.7.7
+Cython==0.29.3
+socketIO-client-nexus==0.7.6
+socketIO-client==0.7.2
 watchdog==0.9.0
-billiard==3.5.0.4
```
Comment (vkotronis) [inline backend/requirements.txt]: What is the dependency that support this?

File: backend/requirements.txt [hunk 2]
```diff
@@ -1,18 +1,10 @@
...
-kombu==4.2.1
 netaddr==0.7.19
-pamqp==1.6.1
```
Comment (vkotronis) [inline backend/requirements.txt]: What is the dependency that support this?

File: backend/requirements.txt [hunk 3]
```diff
@@ -1,18 +1,10 @@
...
-pid==2.2.0
```
Comment (vkotronis) [inline backend/requirements.txt]: What is the dependency that support this?

File: backend/requirements.txt [hunk 4]
```diff
@@ -1,18 +1,10 @@
...
 py-radix==0.10.0
-pytz==2018.5
```
Comment (vkotronis) [inline backend/requirements.txt]: What is the dependency that support this?

File: backend/requirements.txt [hunk 5]
```diff
@@ -1,18 +1,10 @@
...
-PyYAML==4.2b4
```
Comment (vkotronis) [inline backend/requirements.txt]: also this

File: backend/requirements.txt [hunk 6]
```diff
@@ -1,18 +1,10 @@
...
-ruamel.yaml==0.15.47
-setproctitle==1.1.10
```
Comment (vkotronis) [inline backend/requirements.txt]: also this

File: backend/requirements.txt [hunk 7]
```diff
@@ -1,18 +1,10 @@
...
-socketIO-client-nexus==0.7.6
-socketIO-client==0.7.2
-vine==1.1.4
```
Comment (vkotronis) [inline backend/requirements.txt]: also this

File: backend/requirements.txt [hunk 8]
```diff
@@ -1,18 +1,10 @@
-amqp==2.3.2
+kombu==4.2.2.post1
+ruamel.yaml==0.15.87
+redis==3.1.0
```
[the remark this answers is not in the document]
Comment (slowr) [reply backend/requirements.txt]: https://github.com/andymccurdy/redis-py#upgrading-from-redis-py-2x-to-30

**Pull request merged**